README.md

```diff
@@ -120,14 +120,40 @@ const converter = (node, next) => ({
 });
 ```
 
-Skipping an element can be done by returning the result of the `next`-function. Ignoring an element AND its' children can be done by just returning an empty array. **Skipping is the default behavior of any tag that is not supported.**
+Skipping an element can be done by returning the result of the `next`-function. Ignoring an element AND its' children can be done by just returning an empty array.
 
 ```typescript
 const skippingConverter = (node, next) => next(node);
 
 const ignoringConverter = (node, next) => [];
 ```
 
+### Unsupported tags
+
+Skipping an element is the default behavior of any tag that is not supported. However, this can be overridden as follows:
+
+```typescript
+import {
+  DEFAULT_TAG_CONVERTER,
+  htmlStringToDocument,
+  Options,
+  TagConverter,
+} from "contentful-rich-text-html-parser";
+
+const defaultConverter: TagConverter = (node, next) => {
+  console.log(`Unsupported tag: ${node.tagName}`);
+  return next(node); // skip element
+};
+
+const options: Options = {
+  convertTag: {
+    [DEFAULT_TAG_CONVERTER]: defaultConverter,
+  },
+};
```

Comment:
Could we have a separate option named `defaultTagConverter` instead og setting it up with an UUID inside `convertTag`?

Like this:
```typescript
const options: Options = {
  convertTag: {
    ...
  },
  defaultTagConverter: defaultConverter,
};
```

That way we could also set `options.defaultTagConverter` to `convertTagToChildren` in `optionsWithDefaults` to make the code a bit cleaner.

Reply:
Sure! See https://github.com/oleast/contentful-rich-text-html-parser/pull/256/commits/bd710f2b1165a376f3f9a66c7ab913c191954ea7.